src/plugin.scss

```diff
@@ -84,38 +84,18 @@
     overflow: auto;
     position: absolute;
     padding-bottom: 15px;
-    top: 75px;
+    top: 50px;
     left: 15px;
     right: 15px;
     bottom: 61px; /* dialog padding + ok button + 10 */
 }
 
 .vjs-errors-headline {
-    font-size: 22px;
+    font-size: 18px;
     font-weight: bold;
     margin-top: 0;
 }
 
-.vjs-errors-close-button {
```

Comment:
These potentially should be applied to the `.vjs-errors-display .vjs-close-button`. It may help with aligned, not sure.